tools/pkglint/lib/rules.ts

```diff
@@ -1191,6 +1191,53 @@ export class EslintSetup extends ValidationRule {
   }
 }
 
+export class JestSetup extends ValidationRule {
+  public readonly name = 'package-info/jest.config';
+
+  public validate(pkg: PackageJson): void {
+    const cdkBuild = pkg.json['cdk-build'] || {};
+
+    // check whether the package.json contains the jest key,
+    // which we no longer use
+    if (pkg.json.jest) {
```

Comment:
This is not correct, is it? It should be 

```suggestion
    if (cdkBuild.jest) {
```

?

I guess this lint rule is not running on any package now.

Reply:
Can we update the linter so that any package with a top level `jest` block will fail?

Reply:
That's exactly what the line you want to change does...?

Did you not see the comment above?

```ts
    // check whether the package.json contains the jest key,
    // which we no longer use
```

Did you really think I did the changes to ~140 `package.json` files manually? :D No, I ran `pkglint -f` on every package in the `@aws-cdk` directory!

Reply:
Damn! Apparently I can't read code anymore :(